odoo/sql_db.py

```diff
@@ -537,6 +537,8 @@ def rollback(self):
         return result
 
     def __getattr__(self, name):
+        if self._closed and name == '_obj':
+            raise psycopg2.InterfaceError("Cursor already closed")
```

Comment:
Wouldn't it be simpler to assign `self._obj = None` in method `_close()`, instead of removing the attribute?

Reply:
Even with your proposal, you don't want a `NoneType` Error, we want a InterfaceError.
Note: I still modify some part of the last commit.

Reply:
Do we really need to release that object?
Wouldn't it be easier to just leave `self._obj` as it is?